src/sonic-config-engine/tests/simple-sample-graph-case.xml

```diff
@@ -159,7 +159,7 @@
           <Dhcpv6Relays>fc02:2000::3;fc02:2000::4</Dhcpv6Relays>
           <Dhcpv6OptionRfc6939>false</Dhcpv6OptionRfc6939>
```

Comment:
the `Dhcpv6OptionRfc6939` is not in the minigraph, however it will be in the config and is always on.